includes/api/ApiQueryWatchlist.php

```diff
@@ -270,7 +269,7 @@ private function extractRowInfo( $row ) {
 		if ( $this->fld_user || $this->fld_userid ) {
 
 			if ( $this->fld_user ) {
-				$vals['user'] = $row->rc_user_text;
+				$vals['user'] = User::whoIs( $row->rc_user );
```

Comment:
Should we use `User::getUsername` here for fallback?

Reply:
`rc_user_text` column has been removed from the SELECT query - https://github.com/Wikia/app/pull/13879/commits/14581ca643cc5e2c578b5c322892082f9feb0fcd#diff-9dfbf29bb88cc62b4c488c34e2af9aaaL103

Reply:
Good point, added in e60a5f3